schema/method_units/physical/psp/file.json

```diff
@@ -0,0 +1,58 @@
+{
+    "schemaId": "method-physical-psp-file",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "Schema for pseudopotential files",
+    "properties": {
+        "element": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 2
+        },
+        "type": {
+            "description": "Type of pseudopotential",
+            "enum": [
+                "us",
+                "nc",
+                "paw",
+                "coulomb"
+            ]
+        },
+        "format": {
+            "description": "Serialization format for pseudopotential",
+            "enum": [
+                "upf",
+                "vasp"
+            ]
+        },
+        "origin": {
+            "$ref": "file/origin.json"
+        },
+        "storage": {
+            "$ref": "../../../system/storage.json"
+        },
+        "cutoffs": {
+            "$ref": "file/cutoffs.json"
+        },
+        "radii": {
+            "description": "Radius cutoffs",
+            "$ref": "file/radii.json"
+        },
+        "zValence": {
+            "description": "Valence charge, i.e. number of valence electrons",
+            "type": "integer"
+        },
+        "modelReference": {
+            "description": "Unique identifier of the model used to generate the PP set",
+            "type": "string"
+        },
+        "relativity": {
+            "description": "Level of relativistic effects included in the pseudopotential",
+            "enum": [
+                "none",
+                "1c-scalar",
+                "2c-quasi",
+                "4c-full"
```

Comment:
Where did the 1c, 2c, 4c come from? Should we include a link?

Reply:
Speaking about links - should we consider including a DOI for the physical methods like pseudopotential?

Reply:
These slugs refer to 1-component, 2-component, and 4-component relativistic Hamiltonians. On this subject, I find this article very helpful:  https://hal.archives-ouvertes.fr/hal-00662643

Reply:
Regarding the DOIs, we could include the literature reference schema as part of `metadata` for example.

Reply:
Let's have a task to come back to this

Reply:
And keep the link in description (https://hal.archives-ouvertes.fr/hal-00662643)